libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -233,6 +233,32 @@ void GCS_MAVLINK::send_power_status(void)
                                   hal.analogin->power_status_flags());
 }
 
+void GCS_MAVLINK::send_battery_status(const AP_BattMonitor &battery, const uint8_t instance) const
+{
+    float temp;
+    bool got_temperature = battery.get_temperature(temp, instance);
+    mavlink_msg_battery_status_send(chan,
+                                    instance, // id
+                                    MAV_BATTERY_FUNCTION_UNKNOWN, // function
+                                    MAV_BATTERY_TYPE_UNKNOWN, // type
+                                    got_temperature? (temp * 100) : 0x7fff, // temperature. INT16_MAX if unknown
+                                    battery.get_cell_voltages(instance), // cell voltages
```

Comment:
*get_cell_voltages* returns *cell_voltages_t*. I'm not a fan of that typedef, I can live with it if it has some use, which apparently isn't the case as this is relying on it being an array of 10 uint16_t. We should probably have some protection here to make sure the returned array has a size equal or bigger than 10.